### PR TITLE
70 follow software dependency licenses

### PR DESCRIPTION
**Pull request by sudhacheran**
'webpack-license-plugin' configured in 'webpack.config.js' 
This configuration extracts open source license information about all of the dependency packages used in the application and stored it in 'build/third_party_license.json'

### CODE REVIEW

**Review by charliepnnl — DISMISSED** [review]
Approved with some comments that need to be addressed.

In addition:
- Please add a comment saying what testing you have done.
- Please put a copy of the generated JSON file in a PR comment for reference – since it is not part of the commit.
- Please merge main into this branch. This PR should not be modifying root_layout.tsx or templages_config.ts.

File: config/webpack.config.js [hunk 1]
```diff
@@ -587,6 +590,10 @@ module.exports = async function (webpackEnv) {
     },
     plugins: [
       // Generates an `index.html` file with the <script> injected.
+
+      // 'webpack-license-plugin' outputs the dependecy pacakges license information to 'build/third_party_license.json' file.
```
[the remark this answers is not in the document]
Comment (sudhacheran) [reply config/webpack.config.js:594]: updated the comment on this accordingly

File: config/webpack.config.js [hunk 2]
```diff
@@ -587,6 +590,10 @@ module.exports = async function (webpackEnv) {
     },
     plugins: [
       // Generates an `index.html` file with the <script> injected.
```
[the remark this answers is not in the document]
Comment (sudhacheran) [reply config/webpack.config.js:592]: placed the comment above HtmlWebpackPlugin

**Comment by sudhacheran** [comment]
-  Testing: Execute > yarn build
     - all reported prettier errors were cleared in the branch code base
     - the plugin runs and create a file with license details of all the third party packages used in the application under build folder 'third_party_license.json'
- Attached 'third_party_license.json' generated during my local build.
[third_party_license.json.zip](https://github.com/pnnl/Quality-Install-Tool/files/12233556/third_party_license.json.zip)

**Review by charliepnnl — APPROVED** [review]

**Pull request merged**